net-misc/nyx/nyx-2.1.0-r5.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} pypy3 )
+PYTHON_REQ_USE='ncurses,sqlite(-)'
+DISTUTILS_USE_PEP517=setuptools
+
+inherit distutils-r1
+
+DESCRIPTION="Utility to monitor real time Tor status information"
+HOMEPAGE="https://nyx.torproject.org"
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://git.torproject.org/nyx.git"
+	inherit git-r3
+else
+	SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~mips ~ppc ~ppc64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
+	|| ( <net-libs/stem-1.8.0_p20211118[${PYTHON_USEDEP}] ~net-libs/stem-1.8.1[${PYTHON_USEDEP}] )
```

Comment:
It'll be better to just make it depend on >=net-libs/stem-1.8.1 or something instead.

Reply:
The stem versioning is a mess. `stem-1.8.0_p20211118` is actually a git snapshot from the **master** branch and therefore newer than `net-libs/stem-1.8.1` from the **maint** branch.
`stem-1.8.0_p20211118` should probably be `stem-9999_p20211118` and nyx should depend on `<net-libs/stem-9999` since it doesn't work with the current git master branch of stem.

Or we just drop `stem-1.8.0_p20211118` and effectively _downgrade_ `~arch` users to `net-libs/stem-1.8.1`

Reply:
> Oh, I see. Bleh. Can we drop the snapshot as you suggest then?

Added a commit removing it.

> I suppose we may need a comment above RDEPEND explaining the odd structure and what it can be cleaned up into after a while. Could you do that?

Added a comment to RDEPEND